tox.ini

```diff
@@ -1,22 +1,29 @@
+[flake8]
+max-line-length = 100
+ignore = W503
```

Comment:
Do we actually have any W503 violations here? It doesn't look like it at a glance.

Reply:
Not at all, will remove it.